index.html

```diff
@@ -249,6 +250,94 @@ <h2>
           </a>
         </div>
       </section>
+
+      <!-- ---------- GROUPS ---------- -->
```

Comment:
Liked the groups addition! Consider adding a header of some kind to the groups section to help distinguish these from the software.